### PR TITLE
feat: component to display a credential

### PR DESCRIPTION
**Pull request by tarannssingh**
I have created a component to display a password. Currently, I have a hard coded example of a username/password being passed in to the component, which is okay for now as we are still developing features. This is a significant change as it gives us a component to bring our backend functions to life.

### CODE REVIEW

**Comment by Gary055** [comment]
code looks good with little conflicts

File: packages/react-frontend/src/components/create/create.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  } from "@/components/ui/dialog"
+  import {
+    Card,
+    CardContent,
+    CardDescription,
+    CardFooter,
+    CardHeader,
+    CardTitle,
+  } from "@/components/ui/card"
+import { useEffect, useState } from "react";
```
Comment (epears04) [inline packages/react-frontend/src/components/create/create.jsx:17]: Don't seem to use useEffect or useState, import may not be needed

**Review by epears04 — COMMENTED** [review]
Looks pretty good just make sure to clean up some things. Also, please let us know when you connect it to the backend

**Comment by tarannssingh** [comment]
Thank you for the comments. The "import { useEffect, useState } from "react";" line is now removed from that file. I also commented out unused imports that are specific to the component library we are using and will delete them once the component is finalized.

**Comment by zoilak** [comment]
Looks great to me!! Like Ellie mentioned cleaning it up and making sure it follows the flow of prettier

**Pull request merged**